spec/es_spec_helper.rb

```diff
@@ -0,0 +1,27 @@
+require "logstash/devutils/rspec/spec_helper"
+require "ftw"
+require "logstash/plugin"
+require "logstash/json"
+require "stud/try"
+
+
+RSpec.configure do |config|
+  config.before(:all) do
+    require "integration/docker_rspec_helper"
+    @docker = RspecDocker.new
+    @docker.provision("elasticsearch", "spec/integration/docker/Dockerfile")
+    @es_server = {
+      :host => @docker.host[:host_ip],
+      :port => {
+        'http' => @docker.host[:ports]['9200/tcp'][0]['HostPort'],
+        'transport' => @docker.host[:ports]['9300/tcp'][0]['HostPort'],
+        'node' => @docker.host[:ports]['9300/tcp'][0]['HostPort']
+      }
+    }
+    sleep 13
```

Comment:
This 13 here for a reason? can we ping the host to be awake? more on the side of what @untergeek is doing for zabbix.